client/gutenberg/extensions/editor-notes/index.js

```diff
@@ -3,6 +3,7 @@
 /**
  * External dependencies
  */
+import React from 'react';
 import { registerBlockType } from '@wordpress/blocks';
 import { RichText } from '@wordpress/editor';
 
```

Comment:
I believe that the style import below is causing the `Uncaught SyntaxError: Unexpected token .` error, which is currently not present in master.

Reply:
Seems like the stylesheet is loaded as a script:

```
<script src="/calypso/gutenberg-editor.css"></script>
```

We might want to audit our webpack module rules to skip the css files as assets; or modify the `AssetsWriter` to ignore them.